dev/bin/knative-installer.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 knative-setup.sh
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header=$'\e[1;33m'
+  reset=$'\e[0m'
+else
+  header=''
+  reset=''
+fi
+
+# The Knative Core APIs, like Broker or Trigger
+eventing_core_url=https://raw.githubusercontent.com/openshift/knative-eventing/release-v1.1/openshift/release/knative-eventing-ci.yaml
+# Knative Kafka offering: All Kafka centric APIs
+eventing_kafka_url=https://gist.githubusercontent.com/matzew/82280d23baa6cfd6c332c2325f5785c9/raw/848e2d9b36b39c491a60ba24f4ce167201a218ef/knative-kafka.yaml
```

Comment:
Is this yaml static or do we expect it to change?
I would prefer to have this yaml as file in this repo instead of a personal gist

Reply:
something for @matzew

Reply:
@matzew ping

Reply:
@danielezonca  you mean, like checking the manifest into this repo?

sure, I can do that

Reply:
I added a PR for this: https://github.com/r00ta/sandbox/pull/12

Reply:
@manstis see ^^